lib/NativeCode/DynamicLibraryLoaderHelper/DynamicLibraryLoaderHelper.sln

```diff
@@ -31,6 +33,12 @@ Global
 		{251D4477-4C90-4FE1-94A2-52377327D3B2}.Release|x64.Build.0 = Release|x64
 		{251D4477-4C90-4FE1-94A2-52377327D3B2}.Release|x86.ActiveCfg = Release|Win32
 		{251D4477-4C90-4FE1-94A2-52377327D3B2}.Release|x86.Build.0 = Release|Win32
+		{D7462EAF-0024-4EA5-9D89-DBBBC7766333}.Debug|x64.ActiveCfg = Debug|x64
+		{D7462EAF-0024-4EA5-9D89-DBBBC7766333}.Debug|x64.Build.0 = Debug|x64
+		{D7462EAF-0024-4EA5-9D89-DBBBC7766333}.Debug|x86.ActiveCfg = Debug|Win32
+		{D7462EAF-0024-4EA5-9D89-DBBBC7766333}.Debug|x86.Build.0 = Debug|Win32
+		{D7462EAF-0024-4EA5-9D89-DBBBC7766333}.Release|x64.ActiveCfg = Release|x64
+		{D7462EAF-0024-4EA5-9D89-DBBBC7766333}.Release|x86.ActiveCfg = Release|Win32

```

Comment:
Because these .Debug lines have a .Build, should the Release also have a .Build associated with it? Or no, because you would never build this console application and deliver it to people?

Reply:
You hit the nail on the head - we never build this console application nor do we ever deliver it to people. It's for development purposes only.